content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -221,71 +221,107 @@ sysctl --system
 
-and then install CRI-O:
-```shell
-sudo apt-get install cri-o-1.17
+apt-get update
+apt-get install cri-o cri-o-runc
 ```
+
 {{% /tab %}}
 
-{{% tab name="Ubuntu 18.04, 19.04 and 19.10" %}}
+{{% tab name="Ubuntu" %}}
 
-```shell
-# Configure package repository
-. /etc/os-release
-sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O- | sudo apt-key add -
-sudo apt-get update
-```
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
 
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Ubuntu 20.04     | `xUbuntu_20.04`   |
+| Ubuntu 19.10     | `xUbuntu_19.10`   |
+| Ubuntu 19.04     | `xUbuntu_19.04`   |
+| Ubuntu 18.04     | `xUbuntu_18.04`   |
+<br />
+Then, set `$VERSION` to be the cri-o version matching your kubernetes version.
+For instance, if you want to install cri-o 1.18, `VERSION=1.18`.
+We also support pinning to a particular release. To install 1.18.3, `VERSION=1.18:1.18.3`.
+<br />
+
+Then run
 ```shell
-# Install CRI-O
-sudo apt-get install cri-o-1.17
+echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+
+curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | apt-key add -
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -
+
+apt-get update
+apt-get install cri-o cri-o-runc
 ```
 {{% /tab %}}
 
-{{% tab name="CentOS/RHEL 7.4+" %}}
+{{% tab name="CentOS" %}}
 
-```shell
-# Install prerequisites
-curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_7/devel:kubic:libcontainers:stable.repo
-curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}/CentOS_7/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo
-```
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Centos 8         | `CentOS_8`        |
+| Centos 8 Stream  | `CentOS_8_Stream` |
+| Centos 7         | `CentOS_7`        |
+<br />
```

Comment:
Is the `<br>` required to provide spacing between the table and the following text?

Reply:
yeah!